variants/PrimoCore/variant.cpp

```diff
@@ -0,0 +1,144 @@
+ * +------------+------------------+--------+-----------------+
+ * | 10         | 8                |  P008  | USER LED        |
+ * | 11         | 9                |  P025  | RED LED         |
+ * | 12         | 10               |  P026  | GREEN LED       |
+ * | 13         | 11               |  P027  | BLUE LED        |
+ * +------------+------------------+--------+-----------------+
+ * |            | UART             |        |                 |
+ * +------------+------------------+--------+-----------------+
+ * | 14         | TX               |  P011  |                 |
+ * | 15         | RX               |  P012  |                 |
+ * +------------+------------------+--------+-----------------+
+ * |            | I2C              |        |                 |
+ * +------------+------------------+--------+-----------------+
+ * | 16         | SDA              |  P013  |                 |
+ * | 17         | SCL              |  P014  |                 |
+ * | 18         | SDA1(SDA2)       |  P015  |                 |
+ * | 19         | SCL1(SCL2)       |  P016  |                 |
+ * +------------+------------------+--------+-----------------+
+ * |            | INT              |        |                 |
+ * +------------+------------------+--------+-----------------+
+ * | 20         | INT              |  P024  |                 |
+ * | 21         | INT1             |  P020  |                 |
+ * | 22         | INT2             |  P019  |                 |
+ * | 23         | INT3             |  P006  |                 |
+ * | 24         | INT4             |  P007  |                 |
+ * +------------+------------------+--------+-----------------+
+ * |            | TEST POINT       |        |                 |
+ * +------------+------------------+--------+-----------------+
+ * | 25         |                  |  P017  | TP4             |
+ * | 26         |                  |  P018  | TP5             |
+ * +------------+------------------+--------+-----------------+
+ * |            | NFC              |        |                 |
+ * +------------+------------------+--------+-----------------+
+ * | 27         |                  |  P009  | NFC1            |
+ * | 28         |                  |  P010  | NFC2            |
+ * +------------+------------------+--------+-----------------+
+ * |            | 32.768KHz Crystal|        |                 |
+ * +------------+------------------+--------+-----------------+
+ * |            |                  |  P000  | XL1             | 
+ * |            |                  |  P001  | XL2             | 
+ * +------------+------------------+--------+-----------------+
+ */
+
+#include "variant.h"
+
+/*
+ * Pins descriptions
+ */
+//const PinDescription g_APinDescription[]=
+const uint32_t g_ADigitalPinMap[] = 
+{
+  // 0 .. 9 - Digital pins
+  // ----------------------
+  // 0..9
+  31, // { PORT0,  31, PIO_DIGITAL, (PIN_ATTR_DIGITAL|PIN_ATTR_PWM), ADC_A5, PWM0, NOT_ON_TIMER},
```

Comment:
Can we remove the unneeded comments in this file?

Reply:
Sure, just a hangover from Arduino's variant. Will remove all the commented out code.